### ISSUE TITLE
Page contents truncated at OS 2x zoom

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#1049]
All Pages	Magnification 	Page contents is getting truncated with 2x zoom level.	"ENVIRONMENT:
OS | Browser 
macOS Big Sur:11.4|Version 90.0.4430.212 (Official Build) (x86_64)

PREREQUISITE:
Magnification : Setup Zoom on OS level
 
STEPS TO REPRODUCE:
1) Open this Url: https://healthequitytracker.org/
2) Observe the page contents if it's getting truncated with 2x Zoom level.
 
ACTUAL RESULT: Page contents are getting truncated with 2x zoom level.

EXPECTED RESULT: Page contents should not be truncated with 2x zoom level. 

Note: We are proceeding with 2x until more guidance is given."	https://drive.google.com/file/d/1SLg1t-3jdqB8RNnyTK7hXB2OlVyf5Cyi/view?usp=sharing

**Comment by benhammondmusic** [comment]
I was unable to replicate this behavior using MacOS with the OS level Zoom Accessibility settings. Closing
